webui/src/default/page-settings.tsx

```diff
@@ -89,6 +91,95 @@ export default function createPageSettings(theme: Theme, prefersDarkMode: boolea
     const additionalRoutes: React.FunctionComponent = () =>
         <Route path='/about' render={() => <About />} />;
 
+    const headTags: React.FunctionComponent<{title?: string, description?: string, keywords?: string, url?: string, imageUrl?: string, type?: string}> = (props) => {
```

Comment:
I'll simplify the head tags to just a `<title>` tag in the openvsx repo and move these elaborate declarations to the open-vsx.org repo